src/watchdog/observers/kqueue.py

```diff
@@ -15,6 +15,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# The `select` module varies between platforms.
+# mypy may complain about missing module attributes
+# depending on which platform it's running on.
+# The comment below disables mypy's attribute check.
+#
+# type: ignore[attr-defined]
+#
+
```

Comment:
Big Picture: I think this is a macOS/BSD only option?  If so we can encase the whole thing in a platform check instead of ignoring.  I guess perhaps a not-windows-not-linux check would be the applicable one here?

---

Small Picture:  Locally, with the just-released mypy 1.1.1 anyways, I get `src/watchdog/observers/kqueue.py:23: error: type ignore with error code is not supported for modules; use `# mypy: disable-error-code=...`  [syntax]`